system-int-test/src/cmr/system_int_test/utils/ingest_util.clj

```diff
@@ -918,7 +918,7 @@
      (dev-sys-util/reset)
      (set-ingest-umm-version-to-current)
      (when-not (empty? providers)
-      (setup-providers providers options))
+       (setup-providers providers options))
```

Comment:
could you say `(when (seq providers)) ...)` instead?

Reply:
good catch!